test-clients/src/main/java/com/hedera/services/bdd/suites/contract/integration/EvmSuites.java

```diff
@@ -0,0 +1,136 @@
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getContractInfo;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getTxnRecord;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileAppend;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
+import static com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.updateLargeFile;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_DELETED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_DELETED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_SIZE_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_OVERSIZE;
+
+public class EvmSuites extends HapiApiSuite {
+    private final Logger LOGGER = LogManager.getLogger();
+
+    public static void main(String... args) {
+        new EvmSuites().runSuiteAsync();
+    }
+
+    @Override
+    protected List<HapiApiSpec> getSpecsInSuite() {
+        return List.of(
+//                HSCS_EVM_008_SelfDesctructUpdatesHederaAccount(),
+                HSCS_EVM_0014_ContractCreateFailsOnLargeContracts()
+        );
+    }
+
+    // Done
+    private HapiApiSpec HSCS_EVM_008_SelfDesctructUpdatesHederaAccount() {
+        return defaultHapiSpec("HSCS_EVM_008_SelfDestructUpdatesHederaAccount")
+                .given(
+                        fileCreate("bytecode")
+                                .path(FACTORY_SELF_DESTRUCT_CONSTRUCTOR_CONTRACT)
+                )
+                .when(
+                        contractCreate("selfDestroying")
+                                .bytecode("bytecode")
+                                .via("contractCreate")
+                                .hasKnownStatus(SUCCESS)
+                )
+                .then(
+                        getAccountInfo("selfDestroying")
+                                .hasCostAnswerPrecheck(ACCOUNT_DELETED),
+                        getContractInfo("selfDestroying")
+                                .hasCostAnswerPrecheck(CONTRACT_DELETED)
+                );
+    }
+
+    // Validates that ContractCreate fails to create a contract > 24KiB
+    // By uploading a LARGE file with multiple file appends, it is possible
+    // However, trying to do it at 1 tx will result in TRANSACTION_OVERSIZE
+    private HapiApiSpec HSCS_EVM_0014_ContractCreateFailsOnLargeContracts() {
```

Comment:
I think that it would be better to place this test in `ContractCreateSuite.java`